### PR TITLE
Improved README with quicklinks at the top

### PR DESCRIPTION
**Pull request by 610v4nn1**
The documentation link at the end or the README file is difficult to find. I added some quicklinks in a very visible position.
It is not great to see them in the landing page of the documentation but we had plans to evolve that page in a different direction anyway.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -69,22 +76,20 @@ What are you looking for?
...
-    or `using Amazon SageMaker <https://renate.readthedocs.io/en/latest/examples/train_classifier_sagemaker.html>`_.
-* `Documentation <https://renate.readthedocs.io>`_
-* `Supported Algorithms <https://renate.readthedocs.io/en/latest/getting_started/supported_algorithms.html>`_
-* `Experimenting with Renate <https://renate.readthedocs.io/en/latest/benchmarking/index.html>`_
+* Examples:
+    * `Train an MLP locally on MNIST <https://renate.readthedocs.io/en/latest/examples/train_mlp_locally.html>`_
+    * `Train a ResNet on SageMaker <https://renate.readthedocs.io/en/latest/examples/train_classifier_sagemaker.html>`_
+* `Documentation website with API doc and examples <https://renate.readthedocs.io>`_
+* `List of the supported algorithms <https://renate.readthedocs.io/en/latest/getting_started/supported_algorithms.html>`_
+* `How to run continula learning experiments using Renate <https://renate.readthedocs.io/en/latest/benchmarking/index.html>`_
```
Comment (wistuba) [inline README.rst:88]: typo

File: README.rst [hunk 2]
```diff
@@ -69,22 +76,20 @@ What are you looking for?
...
       pip install renate
 
-* Examples
-    We provide examples to train your model
-    `locally <https://renate.readthedocs.io/en/latest/examples/train_mlp_locally.html>`_
-    or `using Amazon SageMaker <https://renate.readthedocs.io/en/latest/examples/train_classifier_sagemaker.html>`_.
-* `Documentation <https://renate.readthedocs.io>`_
-* `Supported Algorithms <https://renate.readthedocs.io/en/latest/getting_started/supported_algorithms.html>`_
-* `Experimenting with Renate <https://renate.readthedocs.io/en/latest/benchmarking/index.html>`_
+* Examples:
```
Comment (wistuba) [inline README.rst:83]: the "what are you looking for" became redundant after the introduction of the quicklinks. drop it altogether?

File: README.rst [hunk 3]
```diff
@@ -23,6 +23,13 @@ The implementation is based on `PyTorch <https://pytorch.org>`_
 and `Lightning <https://www.pytorchlightning.ai>`_ for deep learning, and
 `Syne Tune <https://github.com/awslabs/syne-tune>`_ for hyperparameter optimization.
 
+Quick links
```
Comment (wistuba) [inline README.rst:26]: i would add an installation section here as well. it's very common to have that. just installation followed by pip install renate. currently too hard to find.

File: README.rst (hunk 2 above)
Comment (610v4nn1) [reply README.rst:83]: I think it's still valuable to have the links at the bottom. I would like to keep them.

File: README.rst (hunk 3 above)
Comment (610v4nn1) [reply README.rst:26]: OK, I will add it.

**Review by lballes — APPROVED** [review]

**Pull request merged**